simulation/simulation.py

```diff
@@ -21,16 +20,20 @@
 
 class SimManager:
     def __init__(self, no_spaces, min_spaces_per_lot, max_spaces_per_lot, no_cars,
-                 width, height, parking_lot_seed, car_seed, max_time, app_url):
+                 no_rogues, width, height, parking_lot_seed, car_seed, max_time, app_url):
```

Comment:
Maybe use ```num_rogues``` instead, as ```no_rogues``` sounds like a boolean flag that enables / disables rogues.